jade/distribution.yaml

```diff
@@ -4905,6 +4905,21 @@ repositories:
         release: release/jade/{package}/{version}
       url: https://github.com/tork-a/rtctree-release.git
       version: 3.0.1-0
+  rtmros_common:
+    release:
+      packages:
+      - hrpsys_ros_bridge
+      - hrpsys_tools
+      - openrtm_ros_bridge
+      - openrtm_tools
+      - rosnode_rtc
+      - rtmbuild
+      - rtmros_common
+      tags:
+        release: release/indigo/{package}/{version}
```

Comment:
You can not copy-n-paste the indigo entry. You need to invoke bloom to create the Jade specific release tags and branches.